packages/core/ios/LottieReactNative/ContainerView.swift

```diff
@@ -11,9 +16,11 @@ class ContainerView: RCTView {
     private var colorFilters: [NSDictionary] = []
     private var textFilters: [NSDictionary] = []
     private var renderMode: RenderingEngineOption = .automatic
+    @objc weak var delegate: LottieContainerViewDelegate? = nil
```

Comment:
New delegate so that `LottieAnimationViewComponentView` can send the `animationFinish` event to JS

Reply:
Cool! That was broken for some time I AFAIK.

Reply:
hmm this will only affect the new arch. I tested it on the current arch and it works fine.